Ivan_Shytskyi/Lesson_7/Lesson_7.Homework/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Globalization;
+using System.Net.Security;
+using System.Reflection.Metadata.Ecma335;
+
+Console.WriteLine("Homework");
+bool Compare(string str1, string str2)
```

Comment:
The task was " but do not use build-in method", so you should implement lengh/chars comparison instead of